mod_ci/controllers.py

```diff
@@ -1296,24 +1296,38 @@ def blocked_users():
 
         return redirect(url_for('.blocked_users'))
 
-    # Define removeUserForm processing
-    remove_user_form = RemoveUsersFromBlacklist()
-    if remove_user_form.remove.data and remove_user_form.validate_on_submit():
-        blocked_user = BlockedUsers.query.filter_by(user_id=remove_user_form.user_id.data).first()
-        if blocked_user is None:
-            flash("No such user in Blacklist")
-            return redirect(url_for('.blocked_users'))
+    return{
+        'addUserForm': add_user_form,
+        'blocked_users': blocked_users,
+        'usernames': usernames
+    }
+
+
+@mod_ci.route('/blocked_users/<blocked_user_id>', methods=['GET', 'POST'])
```

Comment:
I'd like to see the use of converters in the variables where possible (refer to https://flask.palletsprojects.com/en/1.1.x/api/#url-route-registrations). It's helpful to reduce invalid input into the variables :)